### PR TITLE
Add quoteAssetPrecision field to exchangeInfo

### PR DESCRIPTION
**Pull request by kshalot**
Changed 2020-04-25 (https://binance-docs.github.io/apidocs/spot/en/#change-log)
Also listed in https://binance-docs.github.io/apidocs/spot/en/#exchange-information.

PR redirected from https://github.com/binance-exchange/binance-official-api-docs/pull/128

### CODE REVIEW

**Comment by phamquangquy92vd** [comment]
17504fa

File: rest-api.md [hunk 1]
```diff
@@ -495,7 +495,8 @@ NONE
       "baseAsset": "ETH",
       "baseAssetPrecision": 8,
       "quoteAsset": "BTC",
```
Comment (aboody8007) [inline rest-api.md:497]: 36LT8441Z1qoRKNVQdRhhaUfCDC5BCARgs

File: rest-api.md [hunk 2]
```diff
@@ -495,7 +495,8 @@ NONE
       "baseAsset": "ETH",
```
Comment (aboody8007) [inline rest-api.md:495]: 0xDd3d94B07565E96A66e6bB25CA232FAba552dC26

File: rest-api.md [hunk 3]
```diff
@@ -495,7 +495,8 @@ NONE
...
       "baseAssetPrecision": 8,
       "quoteAsset": "BTC",
-      "quotePrecision": 8,
```
Comment (aboody8007) [inline rest-api.md:498]: aboody8007

**Review by alisscco — COMMENTED** [review]
Thank you .. 
having a hard Time accessing the funds on binance smarty chain..

**Comment by ttt122** [comment]
Please add funds to my Binance Spot Testnet account:
API Key: <あなたのTestnet API Key>
Thank you!

**Comment by ttt122** [comment]
Hello Binance team,

Please add testnet funds to my Binance Spot Testnet account.

API Key: s3ZL7rq0MHraqIkgteXjgxK4ESD79ez6L03UjQHGЗAhGY9zsDLE7URmfNQRIghkС

Thank you!

**Pull request merged**